### PR TITLE
Week 1

### PR DESCRIPTION
**Pull request by abhijeetd700**
@anilrao316

### CODE REVIEW

File: Week-1/Assignment-2/index.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    </div>
+
+    <div class="footer">
+        <img src="./cattyboard.jpg" class = 'img-bottom' alt="" height="100px" width="100px">
+        <div class="text-content">
+            <p>If It Ain't Love</p>
+            <br/>
+            <p>Catson Derulo</p>
+        </div>
+        <div class="icons-container">
```
Comment (anilrao316) [inline Week-1/Assignment-2/index.html]: plz follow bem for naming css u can refer http://getbem.com/naming/

**Comment by anilrao316** [comment]
Add seperate readme file for web fundamentals assignment

**Comment by anilrao316** [comment]
add screenshot or recording of working assignment

File: Week-1/Assignment-2/index.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        <div class="main-container">
+            <div class="upper-content">
+                <img src="./cattyboard.jpg" alt="" class="img" height="200px" width="200px" />
+                <div class="text-content">
+                    <h2>CattyBoard Top 100 Single Charts (11.06.36)</h2> 
+                    <br/>
+                    <p>Unkown Artist</p>
+                    <p>2016.Charts.100songs</p>
+                    <br/>
+                    <br/>
```
Comment (anilrao316) [inline Week-1/Assignment-2/index.html]: we can remove this multiple br and handle it in css by using margin

File: Week-1/Assignment-2/index.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                      <p>3:34</p>
+                      <p><span class="catty-cloud">CATTY CLOUD SYNC</span></p>
+                    </li>
+                  </ul>
+                <!-- <table id="songs">
+                    <tr>
+                        <td>1. One Dance</td>
+                        <td>Crake feat CatKid & Cyla</td>
+                        <td>2:54</td>
+                        <td class="catty">CATTY CLOUD SYNC</td>
```
Comment (anilrao316) [inline Week-1/Assignment-2/index.html]: remove commented code

**Comment by abhijeetd700** [comment]
@anilrao316  Done necessary changes. Please review it once and let me know if any additional changes are required. Thank you..!

**Comment by anilrao316** [comment]
Screenshots can be added here itself as part of pull request description. no need to push it to the codebase.. thats fine lets take care from next assignment

File: Week-1/Assignment-1/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# How Web Works
+What happens behind the scenes when we type github.com in a browser?
+
+###  1. URL parsing :-
```
Comment (anilrao316) [inline Week-1/Assignment-1/README.md]: Add word document here if any

**Comment by abhijeetd700** [comment]
@anilrao316 Added .docx file for Assignment-1

**Pull request merged**